src/site/resources/styleguides/google-java-style-20180523/javaguide.html

```diff
@@ -541,8 +561,11 @@ <h4 id="s4.6.2-horizontal-whitespace">4.6.2 Horizontal whitespace</h4>
   <li>After <code class="prettyprint lang-java">,:;</code> or the closing parenthesis
   (<code class="prettyprint lang-java">)</code>) of a cast</li>
 
-  <li>On both sides of the double slash (<code class="prettyprint lang-java">//</code>) that
-  begins an end-of-line comment. Here, multiple spaces are allowed, but not required.</li>
+  <li>Between any content and a double slash (<code class="prettyprint lang-java">//</code>) which
+  begins a comment. Multiple spaces are allowed.</li>
```

Comment:
>Between any content and a double slash (//) which begins a comment. Multiple spaces are allowed.
Between a double slash (//) which begins a comment and the comment's text. Multiple spaces are allowed.

They removed "on both sides" that was confusing how to do this.
Please double check how we support space after not before.

Reply:
This is just wording update, there is no functional change here. We're following this rule properly.

#### HorizontalWhitespace.java
```java
/**
 * tesiting google style.
 */
public class HorizontalWhitespace {
  int xyz;       // multiple space between content and double slash.
  int abc; //       multiple space between double slash and comment's text.
  int pqr;       //     testing both.
}
```

#### Output:
```bash
$ java -jar .\checkstyle-10.17.0-all.jar -c .\google_checks.xml .\HorizontalWhitespace.java
Starting audit...
Audit done.
```

Reply:
I agree with @Zopsss. It looks more like a wording update than a functional update. They just made it more explicit.

Reply:
@Zopsss , if we missed this test input please add it our test base.
We should always have real test that proves that we covering wording.
if we can not share link to test, and we need to do such CLI example - it means we have gap in test and we should improve it.

Reply:
allowed becomes:
```
int abc;// should be violation
// no violation as it comment start from line begining
```

Reply:
@Zopsss , please create issue on test improvements and it will resolve this diff

Reply:
Closed/merged